nav2_lifecycle_manager/include/nav2_lifecycle_manager/lifecycle_manager_client.hpp

```diff
@@ -45,7 +45,7 @@ class LifecycleManagerClient
   /**
    * @brief A constructor for LifeCycleMangerClient
    */
-  explicit LifecycleManagerClient(const std::string & name);
+  explicit LifecycleManagerClient(const std::string & name, std::string namespace_ = "");
```

Comment:
Both files need `namespace` not `namespace_` but then this is good to merge

Reply:
`namespace` is reserved in cpp I believe

Reply:
Then use `ns` or something, the `_` is for class members only

Reply:
Oh, yeah please if you you're willing